wavefront/server/background_jobs/rag_ingestion/rag_ingestion/processors/file_processor.py

```
@@ -1,48 +1,56 @@
                 os.unlink(temp_file_path)
 
+    def extract_document_type(self, file_type: str) -> DocumentType:
+        if file_type.startswith('text/plain'):
+            return DocumentType.TEXT
+        if file_type.startswith('image/'):
+            return DocumentType.IMAGE
+        if file_type.startswith('application/'):
+            return DocumentType.PDF
         else:
-            raise ValueError(f'Unsupported file type: {mime_type}')
+            raise ValueError(f'Unsupported file type: {file_type}')
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)